### PR TITLE
EventList Card Operations

### PR DESCRIPTION
**Pull request by Devocolypse**
### Card Operations!
- created two custom components to handle editing and deleting cards for authenticated users, respectively: EditButton and DeleteButton
- installed the capacitor plugin "Dialog" to handle confirmation of deleting cards
- Modified the EventContext and EventList in order to make proper requests and to generate buttons, respectively

### Note:
- there are appear to be merge conflicts with the dev from this branch, some of which are related to package.json and package-lock.json. I suspect that merely means I need to install whatever missing npm packages on LT-CardOps, then commit the changes.

### CODE REVIEW

**Comment by RobbieMac82** [comment]
We're prob going to need @janpevny to take a look at this before we can merge it.

**Comment by Devocolypse** [comment]
Went through and negotiated the merge conflicts, will see what this does on my end.

**Comment by Devocolypse** [comment]
Tabs look awesome!! Now we just need a button to make new events. Clicking on Edit or Delete button works just fine when I'm logged in.

File: codeXplode-capstone-client/src/contexts/EventContext.tsx [hunk 1]
```diff
@@ -3,31 +3,34 @@ import React, { createContext, useEffect, useState } from 'react';
...
-	participants?: string;
-	when: string;
+    _id?: number;
+    title: string;
+    description: string;
+    location?: string;
+    host?: any;
+    cuisine: string;
+    meal: string;
+    mediaCardUrl?: string;
```
Comment (RobbieMac82) [inline codeXplode-capstone-client/src/contexts/EventContext.tsx:13]: Let's keep this as an option to pull images for the event cards

**Review by RobbieMac82 — APPROVED** [review]
great job on the event form functionality!

**Pull request merged**